rom/dev/src/fht.rs

```diff
@@ -37,6 +37,9 @@ pub struct FhtDataStore {
     /// FmcAlias TBS size
     pub fmcalias_tbs_size: u16,
 
+    /// RtAlias TBS size
+    pub rtalias_tbs_size: u16,
```

Comment:
I think you can remove this since ROM won't ever populate this. Can just set rtalias_tbs_size to 0 when constructing the FHT.

Reply:
Done.